.github/workflows/build.yml

```diff
@@ -126,19 +127,21 @@ jobs:
       - name: Run Postinstall Script
         if: steps.cache.outputs.cache-hit == 'true'
         run: npm run postinstall 
-      - name: Build Kirby Cookbook
-        run: npm run dist:cookbook
+      - name: Build Cookbook
+        run: npm run dist:cookbook 
+      - name: Build Storybook
+        run: npm run dist:storybook:designsystem
```

Comment:
I do not think we should spend time parallelising these two tasks as cookbook will be removed in the future.